### ISSUE TITLE
AckedBroadcast with akka-streams 2.0.1

### ISSUE THREAD

**Issue by aomalov** [timcharper/acked-stream#2]
Hi there !

Are you planning to revive the AckedBroadcast component , aligned with new akka-streams release ?
Thanks !

**Comment by timcharper** [comment]
Hi @aomalov ! It was experimental and I decided to drop it (I wasn't sure if anybody used it). It could be resurrected, but when the feature was never complete and supporting `2.0` required substantial changes.

There's the `alsoTo` stream operation you can use to tee a single input to two outputs.

**Comment by aomalov** [comment]
Hi @timcharper  !

No way - the AckedBroadcast was really inspiring. The most consistent FlexiRoute logic implementation so far, to my mind. I learned a lot and created AckedRouter with smart multi-casting logic. 
Recently I've implemented AckedSink which (with a local buffer of Promises) transmits messages over Tcp back and forth. 
Guys at Akka still haven't updated their part of Flexi\* docs - so I'm looking forward to see some good stuff in the nearest weeks (from all of you :)
